static/home/css/style.css

```diff
@@ -565,69 +565,211 @@ a:hover{
 	z-index: 3;
 	width: 100%;
 	height: 100%;
-	background-color: rgba(0,0,0,0.4);
+	background-color: rgba(0,0,0,0.5);
 	padding-top: 100px;
 }
 .don-modal-contents{
 	width: 80%;
 	margin: auto;
-	background-color: #ddd;
-	border: 1px #222 solid;
-	border-radius: 7px;
+	border-radius: 12px;
 	overflow: auto;
 	max-height: 65%;
+	background-color: #fafafa;
+	box-shadow: 5px 10px 18px rgba(0, 0, 0, 0.185);
+	position: relative;
 }
 .don-modal-contents .container{
 	max-width: 90%;
 }
-.don-modal-contents .container h4{
-	margin: 10px 0 5px;
-}
-.don-modal-contents h3{
-	text-transform: uppercase;
-	letter-spacing: 1px;
-	font-weight: 100;
-}
+
 .don-modal-contents .close{
 	position: absolute;
-	top: 100px;
-	right: 10%;
-	float:right;
-	color: #333;
-	font-size: 28px;
+	top: 10px;
+	right: 15px;
+	color: #000000;
+	opacity: 0.3;
+	font-size: 24px;
 	font-weight: bold;
 	cursor: pointer;
-	/*transform: translateX(-13px);*/
-	padding-right: 13px; /*instead of transform property*/
-	/*z-index: 2;*/
-	/*margin-right: 13px; */
+	z-index: 2;
+	height: 36px;
+	width: 36px;
+	border-radius: 50%;
+	margin: 15px 30px;
+	text-align: center;
+	background-color:  rgba(255,255,255);
 }
-.don-box1{
-	background-color: #666;
+
+.don-modal-contents .close:hover{
+	color: #ff8c00;
+	opacity: 1;
+	background-color: rgba(255,255,255,0.1);
 }
-.don-box1 h2{
+
+@media only screen and (max-width: 400px) {
+	.don-modal-contents .close {
+		font-size: 18px;
+		height: 24px;
+	    width: 24px;
+		margin: 15px 1px 15px 2px;
+	    border-radius: 50%;
+	}
+  }
```

Comment:
The position of the close button is getting distorted on small screens. Can you fix that too?